--- src/window.c	2011-03-30 10:22:33.000000000 +0200
+++ src/window.c	2012-03-05 16:46:03.000000000 +0100
@@ -829,16 +829,16 @@
 	break;
     }
 
-    if (w->sizeHints.min_width  == w->sizeHints.max_width &&
+    /*if (w->sizeHints.min_width  == w->sizeHints.max_width &&
 	w->sizeHints.min_height == w->sizeHints.max_height)
 	actions &= ~(CompWindowActionResizeMask	      |
 		     CompWindowActionMaximizeHorzMask |
 		     CompWindowActionMaximizeVertMask |
-		     CompWindowActionFullscreenMask);
+		     CompWindowActionFullscreenMask);*/
 
     if (!(w->mwmFunc & MwmFuncAll))
     {
-	if (!(w->mwmFunc & MwmFuncResize))
+	/*if (!(w->mwmFunc & MwmFuncResize))
 	    actions &= ~(CompWindowActionResizeMask	  |
 			 CompWindowActionMaximizeHorzMask |
 			 CompWindowActionMaximizeVertMask |
@@ -848,7 +848,7 @@
 	    actions &= ~(CompWindowActionMoveMask	  |
 			 CompWindowActionMaximizeHorzMask |
 			 CompWindowActionMaximizeVertMask |
-			 CompWindowActionFullscreenMask);
+			 CompWindowActionFullscreenMask);*/
 
 	if (!(w->mwmFunc & MwmFuncIconify))
 	    actions &= ~CompWindowActionMinimizeMask;
