diff --git a/chrome/common/channel_info_posix.cc b/chrome/common/channel_info_posix.cc
index 88b20489b1b5..59f03d5a8b62 100644
--- a/chrome/common/channel_info_posix.cc
+++ b/chrome/common/channel_info_posix.cc
@@ -56,6 +56,7 @@ std::string GetChannelName() {
 }
 
 std::string GetChannelSuffixForDataDir() {
+  return "-87";
   switch (GetChannel()) {
     case version_info::Channel::BETA:
       return "-beta";
@@ -93,7 +94,7 @@ std::string GetDesktopName(base::Environment* env) {
   std::string name;
   if (env->GetVar("CHROME_DESKTOP", &name) && !name.empty())
     return name;
-  return "chromium-browser.desktop";
+  return "chromium-browser-87.desktop";
 #endif
 }
 #endif  // defined(OS_LINUX) && !defined(OS_CHROMEOS)
