diff --git a/net/proxy_resolution/configured_proxy_resolution_service.cc b/net/proxy_resolution/configured_proxy_resolution_service.cc
index d8d6fe7b4c584..7cecb008770dd 100644
--- a/net/proxy_resolution/configured_proxy_resolution_service.cc
+++ b/net/proxy_resolution/configured_proxy_resolution_service.cc
@@ -342,11 +342,6 @@ GURL SanitizeUrl(const GURL& url) {
   replacements.ClearPassword();
   replacements.ClearRef();
 
-  if (url.SchemeIsCryptographic()) {
-    replacements.ClearPath();
-    replacements.ClearQuery();
-  }
-
   return url.ReplaceComponents(replacements);
 }
 
