--- mpv-0.29.1/audio/out/ao_pulse.c.old	2022-05-26 10:48:57.957582842 +0200
+++ mpv-0.29.1/audio/out/ao_pulse.c	2022-05-26 10:49:20.787611537 +0200
@@ -547,7 +547,7 @@
 }
 
 // Pause the audio stream by corking it on the server
-static void pause(struct ao *ao)
+static void mpv_pause(struct ao *ao)
 {
     cork(ao, true);
 }
@@ -821,7 +821,7 @@
     .get_space = get_space,
     .play      = play,
     .get_delay = get_delay,
-    .pause     = pause,
+    .pause     = mpv_pause,
     .resume    = resume,
     .drain     = drain,
     .wait      = wait_audio,
